config/locales/en/models/claim/offence_details.yml

```diff
@@ -0,0 +1,22 @@
+---
+en:
+  activerecord:
+    errors:
+      models:
+        claim/advocate_claim:
+          attributes:
+            offence:
+              blank: Choose an offence category
+              new_blank: Choose an offence
+
+        claim/advocate_hardship_claim:
+          attributes:
+            offence:
+              blank: Choose an offence category
+              new_blank: Choose an offence
+
+        claim/advocate_interim_claim:
+          attributes:
+            offence:
+              blank: Choose an offence category
+              new_blank: Choose an offence
```

Comment:
whole file should be named after the model `offence` to follow convention.

Also, since they are all the same error messages you can have just one set to go under `claim/base_caim` instead?

```
        claim/base_claim:
          attributes:
            offence:
              blank: Choose an offence category
              new_blank: Choose an offence
```

you could/should also arguably put these directly in `claim.yml`  anyway, as they are not related on the offence model but the claims foreign key attribute of offence

Reply:
to clarify these should go in `claim.yml`. I also think, as a tangent, that the `transfer_details.yml` should actually be in `claim.yml` but that should be done on a separate branch.

Reply:
The idea here is to start separating each form step into its own `yml` file as to not have another monolith like the `en.yml` file

Reply:
> to clarify these should go in `claim.yml`. I also think, as a tangent, that the `transfer_details.yml` should actually be in `claim.yml` but that should be done on a separate branch.

The idea here is to start separating each form step into its own `yml` file as to not have another monolith like the `en.yml` file

Reply:
the convention is to have it by model, and i would keep to that. If i was looking for errors on the claim object or another object i would expect to find them in there. The argument about en.yml does not hold up because that was literally everything, not just models. The file structure could/should reflect the yml structure so you could have all model errors in `models/claim/error.yml` for example. Using an view related naming logic to define model error translations is likely to just get confusing